core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/scroll/ParametersParser.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2018 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.modifiers.scroll;
+
+import com.cognifide.aet.job.api.exceptions.ParametersException;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+
+class ParametersParser {
```

Comment:
I would suggest more specific name, e.g. `ScrollModifierParamsParser`

Reply:
This class is package private, but you are right - `ScrollModifierParamsParser` is more meaningful